libtenzir/src/application.cpp

```diff
@@ -83,6 +83,10 @@ void add_root_opts(command& cmd) {
     "plugins to load at startup; the special values 'bundled' "
     "and 'all' enable autoloading of bundled and all plugins "
     "respectively.");
+  cmd.options.add<caf::config_value::list>(
+    "?tenzir", "disabled-plugins",
+    "plugins and builtins to explicitly disable; use to forbid use of "
+    "operators, connectors, or formats by policy.");
```

Comment:
I'm not sure if this would be the best way to disable operators etc. But it doesn't have to be perfect as we can introduce other ways later, so please feel free to go ahead with this.

Reply:
I chose this because it's the only layer of abstraction that equally works for operators, connectors, and formats. The error messages this causes are suboptimal (e.g., "unknown operator `shell`") but it enabled the use case quickly.

Reply:
Yes, I understand — and this is also nice to have in general.